python/letsql/backends/let/tests/conftest.py

```diff
@@ -1,11 +1,27 @@
+import os
+from contextlib import contextmanager
 from typing import Any
 
 import ibis
 import pandas as pd
 import pandas.testing as tm
 import pytest
+import sqlglot as sg
+import sqlglot.expressions as sge
+from ibis.util import gen_name
 
 import letsql as ls
+import letsql.common.utils.snowflake_utils as SU
```

Comment:
@mesejo this import ultimately requires `snowflake.connector` to be present and will fail if not. should this be restructured so the test suite run doesn't require `snowflake.connector`?

Reply:
Yes, as you mentioned, perhaps we need to create a test matrix for each backend, and create tests only for the snowflake backend